infra/gcb/build_and_run_coverage.py

```diff
@@ -243,13 +253,28 @@ def get_build_steps(project_dir):
   return build_steps
 
 
+# pylint: disable=missing-function-docstring
 def main():
   if len(sys.argv) != 2:
     usage()
 
+  gcp_project = 'oss-fuzz'
```

Comment:
rename these to image_project and base_images_project. also rename any other vars that correspond to these.